pkg/kernel/kernel_test.go

```diff
@@ -459,7 +465,7 @@ func (s *slowExitModule) Run(_ context.Context) error {
 func TestModuleFastShutdown(t *testing.T) {
```

Comment:
should we have a test for the ExitCodeNoForeground case?

Reply:
@j4k4

Reply:
added